test/b2b/sessions.test.ts

```diff
@@ -22,7 +31,7 @@ describe("sessions.get", () => {
           {
             expires_at: "2021-08-30T18:16:53.370383Z",
             last_accessed_at: "2021-08-30T17:16:53.370383Z",
-            member_session_id:
+            member_member_session_id:
```

Comment:
Is this a typo or did we change this field?

Reply:
This is a typo - a little find and replace gone awry
Thanks for the catch!